turbo.json

```diff
@@ -18,6 +17,11 @@
     },
     "lint:tsc": {
       "outputs": []
+    },
+    "@apps/hash-graph#build": {
+      "cache": false,
+      "dependsOn": ["^build"],
```

Comment:
The simplest answer is that currently `^build` does not need to be specified, as the `status` package does not have a build step. If we added a bundling step, it would need to be there (and we'd need to think about how the `build.rs` works at that point as we wouldn't want to directly call `codegen.ts`)

The longer answer is that we're using Turborepo, and we should be appropriately modelling the dependency graph. The `hash-graph` application has local dependencies (`status`), and the build step of the `hash-graph` depends on those dependencies (`status`).

Not appropriately modelling these things is what causes unwanted effects and is why we are currently finding ourselves writing `TURBO_FORCE` so often.

This should not affect graph development for the most part (or at all right now), so I wouldn't worry about it.

It may also be helpful when it comes to CI later on.